go/vt/vttablet/onlineddl/executor.go

```diff
@@ -988,6 +997,14 @@ func (e *Executor) cutOverVReplMigration(ctx context.Context, s *VReplStream, sh
 	if err != nil {
 		return err
 	}
+	// Set large enough `@@lock_wait_timeout` so that it does not interfere with the cut-over operation.
+	// The code will ensure everything that needs to be terminated by `migrationCutOverThreshold` will be terminated.
+	// Setting `lock_wait_timeout` here is done to just ensure MySQL does not interrupt prematurely. It is not
+	// done by means of ensuring cleanup/termination of operations.
```

Comment:
I'm not sure what the `It is not
	// done by means of ensuring cleanup/termination of operations.` part is about?

Reply:
I was trying to explain that `lock_wait_timeout` has no role in the cut-over algorithm. It's not placed here so as to be part of the algorithm. We don't rely on MySQL to e.g. ensure proper rollback. The only reason we put it here is to ensure it is "higher than we'd run into".

Reply:
You can probably remove that whole sentence. The comment already says "The code will ensure ...", which is sufficient explanation.

Reply:
Removed.